macros/commands/get_dangling_tables.sql

```diff
@@ -0,0 +1,60 @@
+{% macro get_dangling_tables(deprecated_models_path=none) %}
+  {% set model_schemas, model_tables = elementary.get_model_schemas_and_tables(deprecated_models_path) %}
+  {% set db_tables = elementary.get_tables_in_schemas(model_schemas) %}
+  {% set rendered_model_tables = [] %}
+  {% for model_table in model_tables %}
+    {% do rendered_model_tables.append(model_table.render()) %}
+  {% endfor %}
+  {% for db_table in db_tables %}
+    {% if db_table.render() not in rendered_model_tables %}
+      {% do print(db_table) %}
+    {% endif %}
+  {% endfor %}
+{% endmacro %}
+
+
+{% macro get_tables_in_schemas(schemas) %}
```

Comment:
Sorry for the nit-picking, but maybe call this `get_all_tables_in_schemas` in order to clarify that this one includes tables that are not in dbt compared to the macro below?